CMSIS/NN/Tests/UnitTest/TestCases/TestData/avgpooling_1/config_data.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2010-2020 Arm Limited or its affiliates. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#pragma once
+// Generated by generate_test_data.py
+#define AVGPOOLING_1_OUT_CH 3
+#define AVGPOOLING_1_IN_CH 3
+#define AVGPOOLING_1_CH_MULT 1  // (Only for depthwise conv)
```

Comment:
is it correct this comment in this file?

Reply:
@gmiodice The comment is auto generated as well :-)

Reply:
Yeah we could improve the generate script in future version, so it just leave out the line completely unless it is for depthwise.